src/com/thunder_cut/server/SyncServer.java

```diff
@@ -23,7 +23,9 @@
 public class SyncServer implements ClientCallback, Runnable {
     private static final int PORT = 3001;
     private ServerSocketChannel server;
-    private final List<ClientInformation> clientGroup;
+    private final List<ClientInfo> clientGroup;
+    private Process process;
+    private final Map<ClientInfo, List<ClientInfo>> clientMap;
```

Comment:
Why do you use this Map?

Reply:
To use this commands -> ignore,  blind
When server make sending Data, Use ScrClient's client list 
Example) A,B,C is client
A ignore B
When A generate sending Data, Use A's client list in map
So A send to A and C
When B generate sending Data, Use B's client list in map
So B send to A, B, C because B don't ignore nobody